src/Stores/Promises/__tests__/selectors.test.js

```diff
@@ -0,0 +1,108 @@
+/* @flow */
+
+import { promisesSelector } from '../selectors';
+import { initialState, politicians, promises } from '../../../utils/testFixtures';
+
+describe('Promises selector', () => {
+  it('loads initial state', () => {
+    expect(promisesSelector(initialState)).toEqual(null);
+  });
+
+  it('Returns null if politicians arent loaded', () => {
+    const loadedState = {
+      ...initialState,
+      promises: {
+        ...initialState.promises,
+        promises: [promises[0]],
+      },
+    };
+    expect(promisesSelector(loadedState)).toEqual(null);
+  });
+
+  it('Returns promises', () => {
+    // TODO use the enriched politician object
```

Comment:
That's been done for most of the test data.
But tests also serve as documentation. And in this case, the politician ID is replaced with the politician object.

In those cases I find it much for readable to have the test data close to the test itself.